vue/src/Tag/TagList.vue

```diff
@@ -258,6 +263,7 @@ export default defineComponent({
   data(): TagListState {
     return {
       hasWriteAccess: Matomo.hasUserCapability('tagmanager_write'),
+      triggerTruncateLength: 40,
```

Comment:
@snake14 This should be 50 ?

Reply:
@AltamashShaikh I tried 50 at first, but I thought that 40 looked better. There was some strange wrapping behaviour when I tried 50.